solr/server/etc/jetty-requestlog.xml

```diff
@@ -30,7 +30,7 @@
         <New class="org.eclipse.jetty.server.AsyncRequestLogWriter">
           <Arg><Property name="solr.log.dir" default="logs"/>/yyyy_mm_dd.request.log</Arg>
           <Set name="filenameDateFormat">yyyy_MM_dd</Set>
-          <Set name="retainDays">90</Set>
+          <Set name="retainDays"><Property name="solr.log.requestlog.retaindays" default="3"/></Set>
```

Comment:
I found it. Default was 90 days, but I set the new default to 3 days to not flood someone's disks now that it becomes default.

I also wrapped the default as a property, so you should be able to (not tested) set `-Dsolr.log.requestlog.retaindays=30` to configure the number of days you want to keep them. This needs to be documented in RefGuide.

Reply:
I updated the refguide docs with the new defaults and system property. I found it strange for the "Request logging" paragraph to live in "[Securing Solr](https://solr.apache.org/guide/8_11/securing-solr.html#request-logging)" page, so I also moved it to "[Configuring logging](https://solr.apache.org/guide/8_11/configuring-logging.html)". Let me know what you think.